relay-common/src/time.rs

```diff
@@ -87,7 +75,7 @@ impl Serialize for UnixTimestamp {
     where
         S: serde::Serializer,
     {
-        serializer.serialize_u64(self.0.as_secs())
+        serializer.serialize_u64(self.as_secs())
```

Comment:
do you want to make a point that we serialize seconds ?
or was this just a quick edit.
it is correct but we could write it more simply as `self.0`

Reply:
Both :) This was just to get this compiling, and using `as_secs` is slightly more explicit than `.0` where you have to go look up what it means.